dashboard/lib/deprecated_level_loader.rb

```diff
@@ -0,0 +1,23 @@
+class DeprecatedLevelLoader
```

Comment:
any reason not to make this a Service object?

Reply:
I don't mind moving it into dashboard/lib/services, but what is the reason for wanting to make it a module?

Reply:
Because it doesn't need to be a class, mostly

Reply:
makes sense. addressed in https://github.com/code-dot-org/code-dot-org/pull/41858